committee-steering/governance/sig-governance.md

```diff
@@ -79,17 +80,17 @@ Subproject contributors (as applicable).
 
 ### Chair
 
-- Number: 2-3
-- Membership tracked in [sigs.yaml]  
-  - If no tech lead role is present, Chair assumes responsibilities from [#tech-lead] section.
+- Number: 2+
+- Membership tracked in [sigs.yaml]
   
-  In addition, run operations and processes governing the SIG:
+In addition, run operations and processes governing the SIG:
```

Comment:
In addition to what?

Reply:
It's poor formatting from before :|

```
- Number: 2-3
- Membership tracked in [sigs.yaml]  
  - If no tech lead role is present, Chair assumes responsibilities from [#tech-lead] section.

  In addition, run operations and processes governing the SIG:
```

It's something I plan to clean up in my next PR.